packages/compiler/src/render3/view/styling_builder.ts

```diff
@@ -277,18 +277,18 @@ export class StylingBuilder {
   }
 
   /**
-   * Builds an instruction with all the expressions and parameters for `elementStyling`.
+   * Builds an instruction with all the expressions and parameters for `styling`.
    *
    * The instruction generation code below is used for producing the AOT statement code which is
    * responsible for registering style/class bindings to an element.
    */
-  buildElementStylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool):
-      Instruction|null {
-    const reference = this._directiveExpr ? R3.elementHostStyling : R3.elementStyling;
+  buildstylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool): Instruction
```

Comment:
Seems like we lost the camel-case in the find-and-replace.

```suggestion
  buildStylingInstruction(sourceSpan: ParseSourceSpan|null, constantPool: ConstantPool): Instruction
```